Add -Wall to build options and drop -03 optimization level in accordance with Debian policy.
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,7 +1,7 @@
 #
 # Makefile for Hydra - (c) 2001-2013 by van Hauser / THC <vh@thc.org>
 #
-OPTS=-I. -O3
+OPTS=-I. -Wall
 # -Wall -g -pedantic
 LIBS=-lm
 DIR=/bin
